burn-book/src/building-blocks/tensor.md

```diff
@@ -114,6 +114,8 @@ Those operations are available for numeric tensor kinds: `Float` and `Int`.
 | `tensor.clamp_min(min)`                          | `torch.clamp(tensor, min=min)`                 |
 | `tensor.clamp_max(max)`                          | `torch.clamp(tensor, max=max)`                 |
 | `tensor.abs()`                                   | `torch.abs(tensor)`                            |
+| `tensor.triu(diagonal)                           | `torch.triu(tensor, diagonal)`                 |
+| `tensor.tril(diagonal)                           | `torch.triu(tensor, diagonal)`                 |
```

Comment:
`torch.triu` -> `torch.tril` for the second row.

Reply:
nice, catch! updated